### ISSUE TITLE
No meshes from Godot 3 conversion to 4 beta 13

### ISSUE THREAD

**Issue by Ogre44** [godotengine/godot#71678]
### Godot version

v4.0.beta.arch_linux [23e10d1]

### System information

Artix Linux 6.1.1-zen1-1-zen Plasma 5.26.4 Ryzen 3 3200G

### Issue description

Non builtin meshes from Godot 3.5.1 project are broken and animations/skeletons are wrong

Godot 4 :
![image](https://user-images.githubusercontent.com/69652964/213463697-eefb8117-89f8-44f7-ac34-a663fe70ccca.png)

Godot 3 : 
![image](https://user-images.githubusercontent.com/69652964/213463804-20c0c193-5f54-4aa2-9f98-c3a5399484ee.png)

### Steps to reproduce

Make a model on Blender
Import to Godot 3.5.1 and save open anyway as scene tscn
Open Godot 4 beta 13 and import 3 project
Open scene to find no mesh in the tscn file

### Minimal reproduction project

N/A

**Comment by Zireael07** [comment]
Make a MRP and attach it.

Any errors in console?

**Comment by Ogre44** [comment]
> Make a MRP and attach it.

There it is
> Any errors in console?

With animations `_set index track 0 is out of bounds` otherwise no errors

**Comment by clayjohn** [comment]
Is this happening after deleting the ``.godot`` folder and reimporting the model? Or are you just opening the project up?

**Comment by Ogre44** [comment]
> Is this happening after deleting the `.godot` folder and reimporting the model? Or are you just opening the project up?

Not sure, I didn't delete anything

**Comment by clayjohn** [comment]
Can you try deleting the ``.godot`` folder and reimporting the model?

**Comment by Ogre44** [comment]
> Can you try deleting the `.godot` folder and reimporting the model?

Done no change, can I reimport a tres mesh file?

**Comment by clayjohn** [comment]
> Done no change, can I reimport a tres mesh file?

No, and I don't think the project converter handles those either. You need to reimport the mesh that you exported from Blender

**Comment by Ogre44** [comment]
Is there other way? I don't really have the Blender files anymore

**Comment by fire** [comment]
Without changing anything, I believe you can export the scene as a gltf from godot 3.

**Comment by Ogre44** [comment]
> Without changing anything, I believe you can export the scene as a gltf from godot 3.

Does this work with animations and skeleton? I tried it just now but the bones are spatials and animations have wrong paths, it works for meshes though

**Comment by IvailoBurov** [comment]
All Godot scenes containing meshes with skeletons are broken for me in Godot 4.x.
It's a terrible compatibility breaker.
